Dockerfile

```diff
@@ -9,17 +9,18 @@ LABEL maintainer="sparklyballs, thelamer"
 
 #Add needed nvidia environment variables for https://github.com/NVIDIA/nvidia-docker
 ENV NVIDIA_DRIVER_CAPABILITIES="compute,video,utility"
-ENV NVIDIA_DRIVER_CAPABILITIES="all"
+ENV NVIDIA_VISIBLE_DEVICES="all"
```

Comment:
See my comment

Reply:
removed the nvidia devices variable, added info to readme